### PR TITLE
perlcritic, readability, sprintf instead of concatenation

### PR DESCRIPTION
**Pull request by bieron**
Hello!

I received Your module via CPAN Pull Request Challenge '15.
Here are some minor (mostly readability) improvements. I'm sorry I wasn't able to help with any real issues, like Windows portability for example, but I am not experienced with XS.

#cpanpr #hacktoberfest

### CODE REVIEW

**Comment by autarch** [comment]
The modules in `inc` are simply copied from other CPAN distros. We don't want to change them.

**Comment by 2shortplanks** [comment]
What @autarch means to say is the modules in `inc` are automatically copied into here as part of the build process;  It's not just we've copied and pasted code, but utility code Dist::Zilla has installed in our distribution.  Code in `inc` isn't installed on the machine when the distribution is installed either - it's code that's actually used during the install / testing process itself.

We typically commit this generated code so we can keep track of what code we're actually releasing to the CPAN in version control, but we're not actually maintaining it ourselves.

**Comment by autarch** [comment]
@2shortplanks, no, that's not right. The modules in `inc` were literally copied from CPAN distros. I'm not 100% sure that's necessary these days with `configure_requires` in the meta-info.

**Comment by bieron** [comment]
Wow, thanks for the fast response! I vaguely suspected `inc` nature. I'd like to know if you'd want to merge my request, provided I removed said existence checks and any changes from `inc`? And if so, do you want a new request, or should I revert said changes in this one?

File: lib/MaxMind/DB/Writer/Serializer.pm [hunk 1]
```diff
@@ -210,22 +210,17 @@ sub _store_data {
...
-my @pointer_thresholds;
-push @pointer_thresholds,
-    {
+my @pointer_thresholds = ( {
     cutoff => 2**11,
     offset => 0,
-    };
-push @pointer_thresholds,
+});
+for (19, 27) {
```
Comment (autarch) [inline lib/MaxMind/DB/Writer/Serializer.pm]: I'm not sure this is any clearer than what it replaces.

**Comment by autarch** [comment]
@bieron - Yes, the other changes look fine (except for the one I just commented on). You can either close this PR and make a new one or rebase your branch with a new set of changes. It's up to you. The first option will be simpler if you're not familiar with git rebasing.

**Comment by autarch** [comment]
Thanks, I ended up merging this manually from the CLI with a few extra tweaks.

**Comment by bieron** [comment]
You're welcome

**Pull request closed without merging**